tests/src/performance/Scenario/JitBench/Utilities/FileTasks.cs

```diff
@@ -178,6 +178,7 @@ public static void DeleteDirectory(string path, ITestOutputHelper output)
                 }
                 try
                 {
+                    SetAttributesNormal(path);
```

Comment:
This was somethign I needed on my machine, where it seems like the permissions are a bit funky. Basically, it wasn't able to delete various directories, so it would crash. Changing the attributes gave the app permission to delete the directory.